jsapp/js/components/table.es6

```diff
@@ -31,6 +31,14 @@ const NOT_ASSIGNED = 'validation_status_not_assigned';
 
 export const SUBMISSION_LINKS_ID = '__SubmissionLinks';
 
+const renderCheckbox = (id, label, isImportant) => {
+  let additionalClass = '';
+  if (isImportant) {
+    additionalClass += 'alertify-toggle-important';
+  }
+  return `<div class="alertify-toggle checkbox ${additionalClass}"><label class="checkbox__wrapper"><input type="checkbox" class="checkbox__input" id="${id}"><span class="checkbox__label">${label}</span></label></div>`;
+};
+
```

Comment:
Instead of copying this function from `jsapp/js/mixins.es6`, please add it to `jsapp/js/utils.es6` as an export, and import it in both `jsapp/js/mixins.es6` and `jsapp/js/components/table.es6` :)